code/modules/mob/living/carbon/alien/humanoid/alien_powers.dm

```diff
@@ -156,7 +156,7 @@ Doesn't work on other aliens/AI.*/
 
 /obj/effect/proc_holder/alien/acid/proc/corrode(atom/target,mob/living/carbon/user = usr)
 	if(target in oview(1,user))
-		if(target.acid_act(200, 100))
+		if(target.acid_act(400, 100))
```

Comment:
undocumented?

Reply:
It was documented in original PR and i left a link for it. But i will document it here, ok